src/test-utils/interfaces.ts

```diff
@@ -19,8 +18,8 @@ export interface TestUtilMethod {
   description?: string;
   returnType?: {
     name: string;
-    type: string;
-    typeArguments: Array<TypeArgument>;
+    type?: string;
+    typeArguments?: Array<TypeArgument>;
```

Comment:
These can be undefined, but because the current code (ab)uses `as any` too much, typecheck does not see it